src/main/java/qna/domain/Answer.java

```diff
@@ -72,4 +75,16 @@ public void toQuestion(Question question) {
     public String toString() {
         return "Answer [id=" + getId() + ", writer=" + writer + ", contents=" + contents + "]";
     }
+
+    public void dealWithAnswerDelete(List<DeleteHistory> deleteHistories) {
+        setDeleted(true);
+        deleteHistories.add(new DeleteHistory(ContentType.ANSWER, getId()
+                , getWriter(), LocalDateTime.now()));
+    }
```

Comment:
```suggestion
    public void dealWithAnswerDelete(List<DeleteHistory> deleteHistories, LocalDateTime now) {
        setDeleted(true);
        deleteHistories.add(new DeleteHistory(ContentType.ANSWER, getId()
                , getWriter(), now));
    }
```
큰 고민거리는 아니지만, 랜덤값이나 시간 같은 데이터를 외부에서 주입받는다면 테스트가 용이해질 것 같아요 :)